site/docs/v1/tech/core-concepts/integration-points.adoc

```diff
@@ -62,6 +63,50 @@ When using OAuth/OIDC, there are multiple kinds of tokens: link:/docs/v1/tech/oa
 
 These tokens can be consumed by APIs or other systems to verify that the holder of the token has been authorized by FusionAuth. link:/learn/expert-advice/tokens/[Learn more about JWTs].
 
+=== JWT Storage
+
+FusionAuth recommends the Authorization Code grant, where there is a server side component which exchanges the one-time use authorization code for an access token. This server side component offers a lot of flexibility when it comes to storing the JWT.
+
+But what should you do with it? After all, an access token is a bearer token, which can't be revoked easily (though see link:/learn/expert-advice/tokens/revoking-jwts[here for strategies]).
+
+This depends on what you are using the token for and your security requirements. The FusionAuth team has outlined a link:/learn/expert-advice/authentication/login-authentication-workflows[number of common flows here].
```

Comment:
> The FusionAuth team has outlined...

This is ok, but I don't know that we regularly use the 3rd person in our docs. 

Can we just say "review this thing" , or use the 1st person? Not a big deal either way I suppose.